### DISCUSSION TITLE
导航栏能设置搜索吗？

### DISCUSSION THREAD

**Discussion by mamashuodedui** [ant-design/pro-components#7722] [Q&A] [answered]
是否有设置能控制导航栏的搜索，对于多级导航栏来说不太好找目标
![image](https://github.com/ant-design/pro-components/assets/16791064/7a66545d-879e-494f-8c4d-5d971f7c3d1d)

**Comment by mamashuodedui** [comment, marked as answer]
@chenshuai2144， 帮忙看下？谢谢

**Reply by chenshuai2144** [reply]
暂时不会更新啦
